fluent-dom/src/overlay.js

```diff
@@ -144,7 +144,7 @@ function sanitizeUsing(translationFragment, sourceElement) {
     // If a child of the same type exists in sourceElement, take its functional
     // (i.e. non-localizable) attributes. This also removes the child from
     // sourceElement.
-    const sourceChild = shiftNamedElement(sourceElement, childNode.localName);
+    const sourceChild = getNamedElement(sourceElement, childNode);
```

Comment:
The reason why this is called `shift...` is that it removes elements from `sourceElement`. I don't think `getNameElement` conveys this meaning well enough.